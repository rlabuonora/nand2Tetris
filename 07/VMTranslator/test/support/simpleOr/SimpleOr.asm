@0
D=A
@SP
A=M
M=D
@SP
M=M+1
@0
D=A
@SP
A=M
M=D
@SP
M=M+1
@SP
M=M-1
@SP
A=M
D=M
@SP
M=M-1
@SP
A=M
D=D|M
D=-D
@SP
A=M
M=D
@SP
M=M+1
@1
D=A
@SP
A=M
M=D
@SP
M=M+1
@SP
M=M-1
@SP
A=M
D=M
@SP
M=M-1
@SP
A=M
D=D|M
D=-D
@SP
A=M
M=D
@SP
M=M+1