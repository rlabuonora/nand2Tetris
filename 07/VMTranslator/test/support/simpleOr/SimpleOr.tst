load SimpleOr.asm,
output-file SimpleOr.out,
compare-to SimpleOr.cmp,
output-list RAM[0]%D2.6.2 RAM[256]%D2.6.2;

set RAM[0] 256,

repeat 30 {
  ticktock;
}
output;

repeat 30 {

  ticktock;
}

output;
