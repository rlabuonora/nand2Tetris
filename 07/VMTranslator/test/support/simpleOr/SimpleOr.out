|  RAM[0]  | RAM[256] |
|     257  |       0  |
|     257  |       1  |
